src/DataProtection/AzureStorage/src/Microsoft.AspNetCore.DataProtection.AzureStorage.csproj

```diff
@@ -1,24 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <Description>Microsoft Azure Blob storage support as key store.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>aspnetcore;dataprotection;azure;blob</PackageTags>
-    <IsPackable>true</IsPackable>
-    <UseLatestPackageReferences>true</UseLatestPackageReferences>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.DataProtection" />
-    <Reference Include="Microsoft.Azure.Storage.Blob" />
```

Comment:
Do we still need the Microsoft.Azure.Storage.Blob dependency e.g. in eng/Versions.props?

Reply:
Yep, I already removed it.